src/d2_docker/config/dhis2-core-start.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Do you think any of these changes could affect IOS or Windows installations?

Reply:
I don't think so, this script runs within the docker, an Alpile Linux that ships with a /bin/bash